actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -644,6 +644,24 @@ def time_zone_options_for_select(selected = nil, priority_zones = nil, model = :
       #   collection_radio_buttons(:post, :author_id, Author.all, :id, :name_with_initial) do |b|
       #      b.label(:"data-value" => b.value) { b.radio_button + b.text }
       #   end
+      #
+      # ==== Gotcha
+      #
+      # The HTML specification says when nothing is select on a collection of radio buttons
+      # web browsers do not send any value to server.
+      # Unfortunately this introduces a gotcha:
+      # if an +User+ model has a +category_ids+ field, and in the form none category is selected no +category_ids+ parameter is sent. So,
```

Comment:
This should be "If a +User+" because the way "user" is pronounced is an exception to the article "an" before a vowel rule because it makes the same sound as "y" in "you".